lib/rbac/filterer.rb

```diff
@@ -279,6 +280,10 @@ def search(options = {})
       return targets, attrs
     end
 
+    def is_sti?(klass)
+      klass.respond_to?(:finder_needs_type_condition?) ? klass.finder_needs_type_condition? : false
```

Comment:
@gtanzillo @jrafanie 

I am just thinking if it is covered also behaviour when you will pass objects with `resource_type` column like

```ruby
[MetricRollup.first,
VmPerformance.first]
```
because as

`ManageIQ::Providers::Redhat::InfraManager.finder_needs_type_condition? => true` is **true**
the
`VmPerformance.finder_needs_type_condition? => false` is **not true**

Reply:
Thanks @lpichler. Let me make a test for that case and see.

Reply:
@gtanzillo what did you find?  Is this a test case we need to worry about?

Reply:
@jrafanie, I discussed it with @yrudman and we added 2 more tests for this case. It turned out to be the same case that @lpichler previously did for `MiqAeDomain`.

Like we discussed yesterday, the case where multiple classes are represented in `targets`, like `[MetricRollup, Vmperformance]`,  and they are not STI would not be supported.

/cc @lpichler, @yrudman

Reply:
That makes sense to me.